### PR TITLE
DOCSP-42316: Stable API

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42316>
Staging - <https://deploy-preview-81--docs-scala.netlify.app/connect/stable-api/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/connect/stable-api.txt [hunk 1]
```diff
@@ -21,4 +21,110 @@ Stable API
...
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``ServerApi`` object and specify a {+stable-api+} version. You must use
+   a {+stable-api+} version defined in the ``ServerApiVersion`` enum.
+#. Construct a ``MongoClientSettings`` object using the ``MongoClientSettings.Builder`` class.
```
Comment (norareidy) [inline source/connect/stable-api.txt:46]: S: [by using](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using)
```suggestion
#. Construct a ``MongoClientSettings`` object by using the ``MongoClientSettings.Builder`` class.
```

File: source/connect/stable-api.txt [hunk 2]
```diff
@@ -21,4 +21,110 @@ Stable API
...
+
+.. literalinclude:: /includes/connect/stable-api.scala
+   :start-after: start-enable-stable-api
+   :end-before: end-enable-stable-api
+   :language: scala
+   :copyable:
+   :dedent:
+
+Once you create a ``MongoClient`` instance with
+a specified API version, all commands you run with the client use the specified
```
Comment (norareidy) [inline source/connect/stable-api.txt:60]: S: add "that"
```suggestion
a specified API version, all commands that you run with the client use the specified
```

File: source/connect/stable-api.txt [hunk 3]
```diff
@@ -21,4 +21,110 @@ Stable API
...
+version. If you must run commands using more than one version of the 
+{+stable-api+}, create a new ``MongoClient``.
+
+Configure the {+stable-api+}
+----------------------------
+
+The following table describes the parameters of the ``ServerApi`` class. You can use these
+parameters to customize the behavior of the {+stable-api+}.
```
Comment (norareidy) [inline source/connect/stable-api.txt:68]: S: I'm not sure if "parameters" is the best word for these. Maybe "options" that you can set by calling methods from the ServerApi class?
```suggestion
The following table describes {+stable-api+} options that you can set by calling methods from the ``ServerApi`` class. You can use these
options to customize the behavior of the {+stable-api+}.
```

File: source/connect/stable-api.txt [hunk 4]
```diff
@@ -21,4 +21,110 @@ Stable API
...
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``ServerApi`` object and specify a {+stable-api+} version. You must use
+   a {+stable-api+} version defined in the ``ServerApiVersion`` enum.
+#. Construct a ``MongoClientSettings`` object using the ``MongoClientSettings.Builder`` class.
+#. Instantiate a ``MongoClient`` using the ``MongoClient.create()`` method and
```
Comment (norareidy) [inline source/connect/stable-api.txt:47]: S: the code example doesn't use the create() method, so I'd just say the constructor
```suggestion
#. Instantiate a ``MongoClient`` using the constructor and 
```

**Review by norareidy — APPROVED** [review]
LGTM with some suggestions

File: source/connect/stable-api.txt [hunk 5]
```diff
@@ -21,4 +21,111 @@ Stable API
...
+MongoDB server that doesn't support the {+stable-api+}. Ensure you're connecting to a
+deployment running {+mdb-server+} v5.0 or later.
+
+Provided apiStrict:true, but the command count is not in API Version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+driver-short+} raises this exception if your ``MongoClient`` runs an operation that
+isn't in the {+stable-api+} version you specified. To avoid this error, use an alternative
+operation supported by the specified {+stable-api+} version, or set the ``strict``
+option to ``False`` when constructing your ``ServerApi`` object.
```
Comment (jyemin) [inline source/connect/stable-api.txt:117]:
```suggestion
option to ``false`` when constructing your ``ServerApi`` object.
```

**Review by jyemin — CHANGES_REQUESTED** [review]
One suggestion, but LGTM

**Review by jyemin — APPROVED** [review]

**Pull request merged**